src/usr/share/batwatch/batwatch.py

```diff
@@ -15,36 +13,31 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-"""
-Handles battery change status monitoring and e-mail notifications for Batwatch.
+"""Handles battery change status monitoring and e-mail notifications for the battery
+monitor.
 """
 
-__author__ = 'Joel Luellwitz, Emily Klapp, and Brittney Scaccia'
+__all__ = ['BatWatch', 'CompositeStatus']
+__author__ = 'Joel Luellwitz, Emily Frost, and Brittney Scaccia'
 __version__ = '0.8'
 
 import logging
+import random
 import time
-import gpgmailmessage
 from pydbus import SystemBus
+import gpgmailmessage
 
 UPOWER_BUS_NAME = 'org.freedesktop.UPower'
-FULLY_CHARGED, CHARGING, DISCHARGING, NO_BATTERY = range(4)
-CHARGE_STATUS_DICT = {
-    0: 'Fully Charged',
-    1: 'Charging',
-    2: 'Discharging',
-    3: 'No Battery'
-}
+NO_BATTERY, DISCHARGING, CHARGING, FULLY_CHARGED = range(4)
+CHARGE_STATUS_LIST = ['No Battery', 'Discharging', 'Charging', 'Fully Charged']
```

Comment:
The thing I don't like about this is that the order of this line has to match the order of the line above it. I'd probably rather have something like:

NO_BATTERY, DISCHARGING, CHARGING, FULLY_CHARGED = range(4)
CHARGE_STATUS_LIST = []
CHARGE_STATUS_LIST.append(NO_BATTERY, 'No Battery')
CHARGE_STATUS_LIST.append(DISCHARGING, 'Discharging')
CHARGE_STATUS_LIST.append(CHARGING, 'Charging')
CHARGE_STATUS_LIST.append(FULLY_CHARGED, 'Fully Charged')

This way, technically the order only has to be maintained on the first line.

Also, I would have an inline comment explaining that the order is important and what it is used for.